package.json

```
@@ -10,7 +10,7 @@
     "services:down": "docker compose -f infra/compose.yaml down",
     "lint:check": "prettier --check .",
     "lint:fix": "prettier --write .",
-    "test": "jest --runInBand",
+    "test": "npm run services:up && concurrently -n next,jest --hide next -k -s command-jest \"next dev\" \"jest --runInBand --verbose\"",
```

Acredito que este código esta ficando gigante acho que seria interessante utilizar [npm hooks](https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts) para separar as coisas 

```suggestion
"pretest": "npm run services:up",
"test": "concurrently -n next,jest --hide next -k -s command-jest \"next dev\" \"jest --runInBand --verbose\"",
```